pingaccess/resource_pingaccess_load_balancing_strategy.go

```diff
@@ -0,0 +1,125 @@
+package pingaccess
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/iwarapter/pingaccess-sdk-go/pingaccess"
+)
+
+func resourcePingAccessLoadBalancingStrategy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourcePingAccessLoadBalancingStrategyCreate,
+		Read:   resourcePingAccessLoadBalancingStrategyRead,
+		Update: resourcePingAccessLoadBalancingStrategyUpdate,
+		Delete: resourcePingAccessLoadBalancingStrategyDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: resourcePingAccessLoadBalancingStrategySchema(),
+	}
+}
+
+func resourcePingAccessLoadBalancingStrategySchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		className: &schema.Schema{
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validateClassNameValue,
+		},
+		configuration: &schema.Schema{
+			Type:             schema.TypeString,
+			Required:         true,
+			DiffSuppressFunc: suppressEquivalentConfigurationDiffs,
+		},
+		name: &schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
+		},
+	}
+}
+
+func resourcePingAccessLoadBalancingStrategyCreate(d *schema.ResourceData, m interface{}) error {
+	svc := m.(*pingaccess.Client).HighAvailability
+	name := d.Get(name).(string)
+	className := d.Get(className).(string)
+	config := d.Get(configuration).(string)
+	var dat map[string]interface{}
+	_ = json.Unmarshal([]byte(config), &dat)
+
+	input := pingaccess.AddLoadBalancingStrategyCommandInput{
+		Body: pingaccess.LoadBalancingStrategyView{
```

Comment:
It's worth implementing a `resourcePingAccessLoadBalancingStrategyData` function similar to:
https://github.com/iwarapter/terraform-provider-pingaccess/blob/master/pingaccess/resource_pingaccess_oauth_server.go#L150

Which returns a LoadBalancingStrategyView so you can remove the duplicated code in the `Create` and `Update` functions.

Reply:
Created resourcePingAccessLoadBalancingStrategyReadData